### PR TITLE
i18n: Spanish normalize_es

### PR DESCRIPTION
**Pull request by SoloVeniaASaludar**
Add "normalize_es", same functionality than existing "normalize_en" (issue #721)

### CODE REVIEW

File: mycroft/util/parse.py [hunk 1]
```diff
@@ -102,3 +107,56 @@ def normalize_en(text, remove_articles):
         normalized += " " + word
 
     return normalized[1:]  # strip the initial space
+
+
+# TODO: it should be modular in indepent files
+# TODO: numbers greaters than 100
```
Comment (penrods) [inline mycroft/util/parse.py]: Typo: "numbers greater than 100"

File: mycroft/util/parse.py [hunk 2]
```diff
@@ -102,3 +107,56 @@ def normalize_en(text, remove_articles):
         normalized += " " + word
 
     return normalized[1:]  # strip the initial space
+
+
+# TODO: it should be modular in indepent files
```
Comment (penrods) [inline mycroft/util/parse.py]: I'm fine with having these in a single file for now.  If the pattern continues for other languages we can implement a single function that splits, removes articles, etc. using just language-specific arrays passed in.

**Comment by penrods** [comment]
This generally looks good to me.  @augustnmonteiro can you give it a once-over?

**Comment by SoloVeniaASaludar** [comment]
Hi,

Before approval of this one, it could be interesting decide what to do with issue #723, related to this code.

Thanks.

**Comment by augustnmonteiro** [comment]
Approved!

**Pull request merged**